armi/reactor/blocks.py

```diff
@@ -1849,6 +1849,10 @@ def getPinToDuctGap(self, cold=False):
                 # getPinCenterFlatToFlat only works for hexes
                 # inner most duct might be circle or some other shape
                 duct = None
+            elif not isinstance(duct, components.HoledHexagon):
```

Comment:
This looks like it won't ignore the holed hexagon still. Maybe this should be `elif isinstance(...)`?